scripts/github-actions/tests-e2e-direct.sh

```diff
@@ -24,23 +24,26 @@ cd ${REPO_ROOT}/
 echo "Downloading envtest assets..."
 export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
 
-#export KCC_USE_DIRECT_RECONCILERS=LoggingLogMetric
+export KCC_USE_DIRECT_RECONCILERS=MonitoringDashboard
 
 echo "Running e2e tests samples for LoggingLogMetric direct reconciliation..."
-
 GOLDEN_OBJECT_CHECKS=1 \
 GOLDEN_REQUEST_CHECKS=1 \
 E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=mock \
   go test -test.count=1 -timeout 600s -v ./tests/e2e -run 'TestAllInSeries/samples/linear-log-metric|TestAllInSeries/samples/exponential-log-metric|TestAllInSeries/samples/int-log-metric|TestAllInSeries/samples/explicit-log-metric'
 
 echo "Running e2e tests fixtures for LoggingLogMetric direct reconciliation..."
-
 GOLDEN_OBJECT_CHECKS=1 \
 GOLDEN_REQUEST_CHECKS=1 \
 E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=mock \
   go test -test.count=1 -timeout 600s -v ./tests/e2e -run 'TestAllInSeries/fixtures/explicitlogmetric|TestAllInSeries/fixtures/exponentiallogmetric|TestAllInSeries/fixtures/linearlogmetric|TestAllInSeries/fixtures/logbucketmetric'
 
-echo "Running scenarios tests for LoggingLogMetric direct reconciliation..."
+echo "Running e2e tests fixtures for MonitoringDashboard direct reconciliation..."
```

Comment:
you may want to specify the `KCC_USE_DIRECT_RECONCILERS` flag because the monitoringDashboard is still using the `dcl2crd` label https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/apis/monitoring/v1beta1/monitoringdashboard_types.go#L870 . Using `KCC_USE_DIRECT_RECONCILERS` flag can force using direct controller.

Reply:
I think I am .... I exported it on line 27.

Reply:
oh, I didn't realize the export there.